source/_integrations/sql.markdown

```diff
@@ -220,6 +220,17 @@ LIMIT
 Replace `-1 day` with the target offset, for example, `-1 hour`.
 Use `state` as column for value.
 
+#### MariaDB
+
+On MariaDB the following where clause can be used to compare the timestamp:
+
+```sql
+...
+  AND last_updated_ts <= UNIX_TIMESTAMP(NOW() - INTERVAL 1 DAY)
+...
+```
+
+Replace `- INTERVAL 1 DAY` with the target offset, for example, `- INTERVAL 1 HOUR`.
 Keep in mind that, depending on the update frequency of your sensor and other factors, this may not be a 100% accurate reflection of the actual situation you are measuring. Since your database won’t necessarily have a value saved exactly 24 hours ago, use “>=” or “<=” to get one of the closest values.
```

Comment:
Uh, that actually got placed a bit unfortunate. The "Keep in mind" part will be part of the MariaDB sub-heading, and it will merge into the same paragraph. :cry: 

````suggestion

Keep in mind that, depending on the update frequency of your sensor and other factors, this may not be a 100% accurate reflection of the actual situation you are measuring. Since your database won’t necessarily have a value saved exactly 24 hours ago, use “>=” or “<=” to get one of the closest values.

#### MariaDB

On MariaDB the following where clause can be used to compare the timestamp:

```sql
...
  AND last_updated_ts <= UNIX_TIMESTAMP(NOW() - INTERVAL 1 DAY)
...
```

Replace `- INTERVAL 1 DAY` with the target offset, for example, `- INTERVAL 1 HOUR`.
````